bcgov/cas-registration
# Feat/309 activity carbonate use

Addresses bcgov/cas-reporting#309. This adds the _Carbonate Use_ activity.

## Changes 🚧

- Adds backend config/migration for _Carbonate Use_ activity.
- Adds frontend page and components for _Carbonate Use_ activity.

## To test 🔬

- `make reset_db` to apply latest migrations.
- spin up the front, backend and postgres.
- Login, then navigate to `http://localhost:3000/reporting/reports/1/facilities/1/activities/carbonates_use`, click through the available form fields. 
  - The `Methodology` dropdown options should spawn further form fields.
  - The `Carbonate Type` dropdown options do not create additional form fields.
